amplify/backend/function/wandbHistory/src/utils/script.py

```diff
@@ -78,3 +78,15 @@ def filter_recent_data(data):
         iterator = filter(lambda ele: (now - datetime.datetime.fromtimestamp((ele["timestamp"]/1000))).days<=3, data)
         filtered_data = list(iterator)
     return filtered_data
+
+def smooth_data(data):
+    df = pd.DataFrame(data)
+    grouped_data = df.groupby('key')
+    # https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.rolling.html
+    df['smoothed_best_average_loss'] = grouped_data['best_average_loss'].rolling( 
+        window=40, # Window size.

```

Comment:
<img width="458" alt="image" src="https://github.com/surcyf123/pretraining-frontend/assets/25290212/a11ef260-13be-416d-85d3-3e4f1c2288b0">

This  is added due to series loss
<img width="465" alt="image" src="https://github.com/surcyf123/pretraining-frontend/assets/25290212/29fdb9dd-e76f-4e27-8779-2dd8843ec9cb">